index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+   
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <main id="main">
+
+   <h1>Dr. Norman Borlaug</h1>
+   <div id="img-div">
+
+    <img src="dr.png" alt="" id="img">
+    <legend id="tribute-info">This is the picture of our doctor</legend>
+    <a href="" id="tribute-link" target="_blank"></a>
+
+   </div>
```

Comment:
I think it would be better to use an indentation of 2  in your HTML file . A good Indentation makes your work looks good and is also best practice.